guides/common/modules/ref_overview-of-authentication-methods-in-project.adoc

```diff
@@ -1,9 +1,15 @@
-[id="overview-of-authentication-methods-in-project_{context}"]
+[id="overview-of-authentication-methods-in-{project-context}_{context}"]
+ifeval::["{context}" == "authentication"]
 = Overview of authentication methods in {Project}
+endif::[]
+ifeval::["{context}" == "planning"]
+= Configuring external authentication in {Project}
+endif::[]
```

Comment:
Not a fan of this construct. I hightly believe that this would break this for me in downstream docs. not a blocker as I can always create a follow-up PR to change this further.

Reply:
It shouldn't break if you define the context. Can you test it first, please?

Reply:
Is it the ifeval itself that's an issue, or just using it for a heading right after defining the anchor? We are using ifevals elsewhere already, for example https://github.com/theforeman/foreman-documentation/blob/e5f1b864dc4eadc13bec2e7b245833da211932d8/guides/common/modules/con_remote-execution-in-project.adoc?plain=1#L35

To be honest, I wasn't totally sure about using ifevals to differentiate headings like this. Won't it be confusing to users if they come across a module that's reused but with a different heading? I think I'll just try to find a heading that will work in both contexts.

Reply:
@maximiliankolb and I agreed that we will keep this as is for now and submit a follow-up PR if it breaks anything